skyportal/handlers/api/source.py

```diff
@@ -1489,9 +1489,7 @@ async def get(self, obj_id):
               application/json:
                 schema: Error
         """
-        source = Obj.get_if_accessible_by(obj_id, self.current_user)
-        if source is None:
-            return self.error('Source not found', status=404)
+        source = Obj.get_if_accessible_by(obj_id, self.current_user, raise_if_none=True)
```

Comment:
Why is this better than raising the 404 explicitly?

Reply:
checking manually is fine, but we're largely moving over to the more compact pattern here. this is changed to easily test via the new source API test

Reply:
OK, automating that makes sense.  Thanks for explaining to me.